src/coreclr/clrfeatures.cmake

```diff
@@ -34,12 +34,18 @@ if(NOT DEFINED FEATURE_DBGIPC)
   endif()
 endif(NOT DEFINED FEATURE_DBGIPC)
 
+string(TOUPPER ${CMAKE_BUILD_TYPE} UPPERCASE_CMAKE_BUILD_TYPE)
+
 if(NOT DEFINED FEATURE_INTERPRETER)
   if(CLR_CMAKE_TARGET_ANDROID)
      set(FEATURE_INTERPRETER 0)
   else()
       if(CLR_CMAKE_TARGET_ARCH_AMD64 OR CLR_CMAKE_TARGET_ARCH_ARM64)
-        set(FEATURE_INTERPRETER $<IF:$<CONFIG:Debug,Checked>,1,0>)
+        if(UPPERCASE_CMAKE_BUILD_TYPE STREQUAL DEBUG OR UPPERCASE_CMAKE_BUILD_TYPE STREQUAL CHECKED)
```

Comment:
This change is incorrect. We cannot check the build type using "if" for multi-target generators. It needs to stay as a generator expression. Also, why have you added the conversion to uppercase? We use the Pascal casing everywhere.